packages/enzyme-test-suite/test/RSTTraversal-spec.jsx

```diff
@@ -97,71 +68,59 @@ describe('RSTTraversal', () => {
       expect(nodeHasProperty(node, 'title')).to.equal(false);
     });
 
-    it('should parse false as a literal', () => {
+    it('should parse false', () => {
       const node = $(<div foo={false} />);
 
-      expect(nodeHasProperty(node, 'foo', 'false')).to.equal(true);
+      expect(nodeHasProperty(node, 'foo', false)).to.equal(true);
     });
 
-    it('should parse false as a literal', () => {
-      const node = $(<div foo />);
-
-      expect(nodeHasProperty(node, 'foo', 'true')).to.equal(true);
```

Comment:
should this test not still be valid for `true` instead of `'true'`?

Reply:
It should, I'll add that case.

Reply:
Added test case for booleans: https://github.com/airbnb/enzyme/pull/1086/files#diff-67b0df7735c607bdef49d8a20292e57aR71